.github/workflows/build.yml

```diff
@@ -0,0 +1,11 @@
+on: [push, pull_request]
+name: build
+jobs:
+  lint:
```

Comment:
Should be called `luacheck` (see https://github.com/mt-mods/plantlife_modpack/pull/53#discussion_r1270833014)